change/@microsoft-fast-foundation-377b942b-ea56-4464-beeb-d6bb80232402.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Using a manual focus to workaround delegatesFocus issues on Firefox",
```

Comment:
If we aren't going to address the issue holistically across the repo, can you scope the message here to just the components touched as part of this exercise?